pkg/tfgen/installation_docs.go

```diff
@@ -372,3 +372,24 @@ func getDefaultHeadersToSkip() []*regexp.Regexp {
 	}
 	return defaultHeaderSkipRegexps
 }
+
+func getTfVersionsToRemove() []*regexp.Regexp {
+	tfVersionsToRemove := []*regexp.Regexp{
+		regexp.MustCompile(`It requires [tT]erraform [v0-9]+\.?[0-9]?\.?[0-9]? or later.`),
+		regexp.MustCompile(`(?s)(For )?[tT]erraform [v0-9]+\.?[0-9]?\.?[0-9]? and (later|earlier):`),
+	}
+	return tfVersionsToRemove
+}
+
+func removeTfVersionMentions(docFile string) tfbridge.DocsEdit {
+	return tfbridge.DocsEdit{
+		Path: docFile,
+		Edit: func(_ string, content []byte) ([]byte, error) {
+			for _, tfVersion := range getTfVersionsToRemove() {
```

Comment:
This compiles the regular expressions in the `.Edit` call, which is not efficient.
```suggestion
	tfVersionsToRemove := getTfVersionsToRemove()
	return tfbridge.DocsEdit{
		Path: docFile,
		Edit: func(_ string, content []byte) ([]byte, error) {
			for _, tfVersion := range tfVersionsToRemove {
```